src/FluentMigrator.Runner.Firebird/Generators/Firebird/FirebirdGenerator.cs

```diff
@@ -39,23 +40,26 @@ public class FirebirdGenerator : GenericGenerator
         protected readonly FirebirdTruncator truncator;
 
         public FirebirdGenerator(
-            [NotNull] FirebirdOptions fbOptions)
-            : this(fbOptions, new OptionsWrapper<GeneratorOptions>(new GeneratorOptions()))
+            [NotNull] FirebirdOptions fbOptions,
+            [NotNull] IOptions<QuoterOptions> quoterOptions)
+            : this(fbOptions, quoterOptions, new OptionsWrapper<GeneratorOptions>(new GeneratorOptions()))
         {
         }
 
         public FirebirdGenerator(
             [NotNull] FirebirdOptions fbOptions,
+            [NotNull] IOptions<QuoterOptions> quoterOptions,
             [NotNull] IOptions<GeneratorOptions> generatorOptions)
-            : this(new FirebirdQuoter(fbOptions.ForceQuote), fbOptions, generatorOptions)
+            : this(new FirebirdQuoter(fbOptions, quoterOptions), fbOptions, quoterOptions, generatorOptions)
         {
```

Comment:
It seems like you moved the ForceQuote flag off of FirebirdOptions and onto QuoterOptions - but I don't see where?  Was it in a previous commit?

Reply:
No, I haven't moved `_forceQuote`, is it in the `FirebirdQuoter` were it had been before I pushed my changes.
The reason for this change is that I simply wanted to use another constructor where `FirebirdOptions` and `IOptions<QuoterOptions>` are provided.
I use this constructor. I does exactly the same - pass `ForceQuote` from `ForceQuote` to `_forceQuote`.
In theory, it can still be rolled back to `new FirebirdGenerator(fbOptions.ForceQuote, new OptionsWrapper ...)`
![image](https://user-images.githubusercontent.com/47413092/118362515-7b1bad80-b598-11eb-8430-392c356e044e.png)